README.md

```diff
@@ -16,31 +16,54 @@ The **Platform.sh CLI** is the official command-line interface for [Platform.sh]
   * [Drush](https://github.com/drush-ops/drush) (for Drupal projects)
   * Other build tools: [npm](https://www.npmjs.com/), [pip](http://docs.python-guide.org/en/latest/starting/installation/), [bundler](http://bundler.io/), etc.
 
-## Installation
+## Install
```

Comment:
we should probably remove the entire Requirements section above, and put a short summary of it under "Legacy installer"?

Reply:
:+1: I'll add a summary in the Legacy installer section.